unit/temporal-logic/trivial_sva.cpp

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************\
+
+Module: Trivial SVA
+
+Author: Daniel Kroening, dkr@amazon.com
+
+\*******************************************************************/
+
+//#include <util/arith_tools.h>
+//#include <util/mathematical_types.h>
+
+//#include <temporal-logic/ltl.h>
```

Comment:
I'd suggest to remove those - they are easy enough to add back in when actually required.